tests/suite/test_dos.py

```diff
@@ -430,8 +434,8 @@ def test_dos_arbitrator(
             wait_before_test()
 
         print("------------------------- Check if new pod receive info from arbitrator -----------------------------")
-        print("Wait for 30 seconds")
-        wait_before_test(30)
+        print("Wait for 60 seconds")
```

Comment:
What's the reason for increasing this wait?

Reply:
When I perform the tests, I saw that in some tests the wait for 30 sec isn't enough, when changed it to 60 the test pass all the time